### ISSUE TITLE
Web incorrectly sets aria-live attribute

### ISSUE THREAD

**Issue by TheOneWithTheBraid** [flutter/flutter#87939]
## Steps to Reproduce

1. Create an empty flutter project
2. Build for web
3. Search for `flt-semantics-placeholder` node in rendered DOM

**Expected results:**
The `aria-live` attribute should be set to `polite`, `assertive` or `off`, according to [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).

I think, `polite` should be what we need.

**Actual results:**
The `aria-live` attribute is set to `true`

<details>
  <summary>Logs</summary>

```
$ flutter analyze
Analyzing test_app...                                                   
No issues found! (ran in 4.3s)
```

```
$ flutter doctor -v
[✓] Flutter (Channel beta, 2.4.0-4.2.pre, on Arch Linux 5.13.5-zen1-1-zen, locale en_US.UTF-8)
    • Flutter version 2.4.0-4.2.pre at /opt/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision f18b9281c2 (2 weeks ago), 2021-07-22 14:08:30 -0700
    • Engine revision 844c29f42a
    • Dart version 2.14.0 (build 2.14.0-301.2.beta)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /opt/android-sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = /opt/android-sdk
    • Java binary at: /usr/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.12+7)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • CHROME_EXECUTABLE = /usr/bin/chromium

[✓] Linux toolchain - develop for Linux desktop
    • clang version 12.0.1
    • cmake version 3.21.0
    • ninja version 1.10.2
    • pkg-config version 1.7.3

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/linux#android-setup for detailed instructions).

[✓] IntelliJ IDEA Community Edition (version 2021.1)
    • IntelliJ at /usr/share/idea
    • Flutter plugin version 57.0.5
    • Dart plugin version 211.7233

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Arch Linux 5.13.5-zen1-1-zen
    • Chrome (web)    • chrome • web-javascript • Chromium 91.0.4472.164 Arch Linux

! Doctor found issues in 1 category.

```

</details>

**Comment by TahaTesser** [comment]
Hi @TheOneWithTheBraid 
Thanks for filing the issue
Can you elaborate on step 3, I am not familiar with it 
Thank you

**Comment by TheOneWithTheBraid** [comment]
![image](https://user-images.githubusercontent.com/29313398/128833829-a34026d1-ef73-4caf-bf0f-169a2b3c9212.png)

If you open the (built) web application and go through the Document Object Model, you will (or will not) find an HTML custom element of the type `flt-semantics-placeholder`.

This element has an invalid ARIA value in it's `aria-live` attribute.

As you can see, Flutter currently renders `aria-live="true"` which is an invalid value as it's not {`polite`, `assertive`, `off`}. For the use case we have in any web application, supporting at least some basic accessibility, this value should be set to `polite`. The detailed specifications of these attributes can be found in the [corresponding MDN page](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).

**Comment by TahaTesser** [comment]
HI @TheOneWithTheBraid 
Ahhh...that's it. I can reproduce it, thanks

|  `stable`  | `master` |
| :---: |:---: |
|  ❌  |   ❌  | 

Check `flutter doctor -v` outputs for each channel below

<details> 
<summary>flutter doctor -v</summary> 
	
```console
[✓] Flutter (Channel stable, 2.2.3, on Linux, locale en_US.UTF-8)
    • Flutter version 2.2.3 at /home/taha/Code/flutter_stable
    • Framework revision f4abaa0735 (6 weeks ago), 2021-07-01 12:46:11 -0700
    • Engine revision 241c87ad80
    • Dart version 2.13.4

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /home/taha/Code/android-sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_SDK_ROOT = /home/taha/Code/android-sdk
    • Java binary at: /home/taha/Code/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • Ubuntu clang version 12.0.0-3ubuntu1~21.04.1
    • cmake version 3.18.4
    • ninja version 1.10.1
    • pkg-config version 0.29.2

[✓] Android Studio (version 2020.3)
    • Android Studio at /home/taha/Code/android-studio
    • Flutter plugin version 58.0.2
    • Dart plugin version 203.8292
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.58.2)
    • VS Code at /usr/share/code
    • Flutter extension version 3.25.0

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Linux
    • Chrome (web)    • chrome • web-javascript • Google Chrome 92.0.4515.107

• No issues found!

```

```console
[✓] Flutter (Channel master, 2.5.0-6.0.pre.40, on Pop!_OS 21.04 5.11.0-7620-generic, locale en_US.UTF-8)
    • Flutter version 2.5.0-6.0.pre.40 at /home/taha/Code/flutter_master
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 7d76a827c0 (10 hours ago), 2021-08-09 15:21:17 -0700
    • Engine revision 4fef55db10
    • Dart version 2.14.0 (build 2.14.0-385.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /home/taha/Code/android-sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_SDK_ROOT = /home/taha/Code/android-sdk
    • Java binary at: /home/taha/Code/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at google-chrome

[✓] Linux toolchain - develop for Linux desktop
    • Ubuntu clang version 12.0.0-3ubuntu1~21.04.1
    • cmake version 3.18.4
    • ninja version 1.10.1
    • pkg-config version 0.29.2

[✓] Android Studio (version 2020.3)
    • Android Studio at /home/taha/Code/android-studio
    • Flutter plugin version 58.0.2
    • Dart plugin version 203.8292
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.58.2)
    • VS Code at /usr/share/code
    • Flutter extension version 3.25.0

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Pop!_OS 21.04 5.11.0-7620-generic
    • Chrome (web)    • chrome • web-javascript • Google Chrome 92.0.4515.107

• No issues found!

```
		
</details>

✅: No Issue ❌: Issue reproduced

**Comment by TheOneWithTheBraid** [comment]
This issue seems to be caused by https://github.com/flutter/engine/blob/66ae7a368c7ec7819a52088009169ef9a9515379/lib/web_ui/lib/src/engine/semantics/semantics_helper.dart#L192. If you don't mind, I will fix it.

**Comment by TheOneWithTheBraid** [comment]
I am quite confused that https://github.com/flutter/engine/blob/8f272d24c120bee4eb6df05258b3f741beff8434/lib/web_ui/test/engine/semantics/accessibility_test.dart#L49 test passes.

**Comment by TahaTesser** [comment]
@TheOneWithTheBraid 
Thanks for the PR!

**Comment by TahaTesser** [comment]
@yjbanov 
https://github.com/flutter/engine/pull/27985 can be closed

**Comment by yjbanov** [comment]
> @yjbanov
> [flutter/engine#27985](https://github.com/flutter/engine/pull/27985) can be closed

Yep! Thanks for the heads-up. Wasn't paying attention.

**Comment by yjbanov** [comment]
@TheOneWithTheBraid thank you for reporting and fixing the issue! I merged your PR.
